imagersite/imager_profile/tests.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+from .models import ImagerProfile
+import factory
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+
+    username = 'badass'
+    email = factory.LazyAttribute(
+        lambda a: '{}@example.com'.format(a.username).lower()
+    )
+
+
+# Create your tests here.
+class UserTest(TestCase):
+    def setUp(self):
+        self.user1 = UserFactory()
+        self.user1.set_password('abc')
+        self.user1.save()
+        self.profile1 = self.user1.profile
+
+    # Test 1
+    # Check that User works as we expect as base class
+    def test_create_user(self):
+        expected = [
+            'badass',
+            'badass@example.com'
+        ]
+        actual = [
+            self.user1.username,
+            self.user1.email
+        ]
+        self.assertEqual(actual, expected)
+        self.assertIsInstance(self.user1.profile, ImagerProfile)
+
+    # Test 2
+    # Check that an imagerProfile is created when user is created
+    def test_create_imagerprofile(self):
+        self.assertEqual(self.profile1.user.username, 'badass')
+
+    # Test 3
+    # Check that camera field can optionally hold data
+    def test_camera_field(self):
+        self.assertFalse(self.profile1.camera)
+        self.profile1.camera = 'potato'
+        self.assertEqual(self.profile1.camera, 'potato')
```

Comment:
This type of test is not of much value.  what might be of more value is ensuring that restrictions on your fields actually work.  Are required fields required?  If you set a length limit on a char field, how does it fail to set that field to a value larger than should be accepted?  These are the types of tests that cover your code.